### PR TITLE
Fix activity indicator with delayed animations on some shape

### PR DESCRIPTION
**Pull request by phimage**
Trying to add a new activity indicator named `rupee` 

ezgif com-video-to-gif 1

and comparing the animation gif in doc and the current one in demo app, I see an issue :
some animation which use `beginTime` doesn't working

I see for working animation that the animation group is created only one time, not for each additional layer. So I edit the bugged ones.

### CODE REVIEW

File: Sources/ActivityIndicators/Animations/ActivityIndicatorAnimationRupee.swift [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+//  Copyright © 2018 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+public class ActivityIndicatorAnimationRupee: ActivityIndicatorAnimating {
+
+  // MARK: Properties
+
+  fileprivate let duration: CFTimeInterval = 1
```
Comment (JakeLin) [inline Sources/ActivityIndicators/Animations/ActivityIndicatorAnimationRupee.swift:15]: We can use `private` now

File: Sources/ActivityIndicators/Animations/ActivityIndicatorAnimationBallSpinFadeLoader.swift [hunk 2]
```diff
@@ -55,7 +56,7 @@ public class ActivityIndicatorAnimationBallSpinFadeLoader: ActivityIndicatorAnim
 
 private extension ActivityIndicatorAnimationBallSpinFadeLoader {
 
-  var animation: CAAnimationGroup {
+  var defaultAnimation: CAAnimationGroup {
```
Comment (JakeLin) [inline Sources/ActivityIndicators/Animations/ActivityIndicatorAnimationBallSpinFadeLoader.swift:59]: Great fix 👍

**Review by JakeLin — APPROVED** [review]
Great animation 👍, can you also add it to the domo app?

File: Sources/ActivityIndicators/Animations/ActivityIndicatorAnimationRupee.swift (hunk 1 above)
Comment (phimage) [reply Sources/ActivityIndicators/Animations/ActivityIndicatorAnimationRupee.swift:15]: Could be done later. I just copy paste.
I want to make it public, editable later.
I try to add this duration as protocol parameter, but failed, some must total animation duration must be calculated

**Comment by phimage** [comment]
Demo app is updated automatically using enum ;)

**Comment by JakeLin** [comment]
@phimage But I can't find the animation `rupee` in the demo app playground 🤔

**Comment by phimage** [comment]
@JakeLin did you build the framework?
Then build and launch the demo app?

I do not know why and since when but I see no target or compile dependencies between projects

screen shot 2018-05-07 at 15 01 01
screen shot 2018-05-07 at 15 01 07

**Comment by JakeLin** [comment]
@phimage thanks, you are absolutely right, I can see it now. I removed the dependence by accident when I upgraded the project to support Xcode 9.3. Can you add it back to this PR? Or I can do a comment in master later on?

**Comment by phimage** [comment]
@JakeLin  I merge so you can edit main branch (or make new PR)

**Pull request merged**